i18n/ru.json

```diff
@@ -0,0 +1,155 @@
+{
+	"global.back":							"Назад",
+	"global.done":							"Выполнить",
+	"global.qr-code":						"QR-код",
+	"global.save":							"Сохранить",
+	"global.update":						"Обновить",
+	"global.ok":							  "OK",
+	"global.cancel":						"Отмена",
+	"global.advanced.show":			"Открыть расширенные опции",
+	"global.advanced.hide":			"Закрыть расширенные опции",
+
+	"memo.memo":							  "Памятка",
+	"memo.type":							  "Тип памятки",
+	"memotype.none":						"Пусто",
+	"memotype.id":							"ID",
+	"memotype.text":						"Текст",
+	"memotype.hash":						"Хэш",
+	"memotype.return":					"Вернуться",
+
+	"error.transaction-failed":	"Перевод не состоялся",
+	"error.missing-signatures":	"Не достаточно подписей",
+
+	"tabs.home":							  "Обзор",
+	"tabs.home.pull":						"Коснитесь, чтобы обновить...",
+	"tabs.home.activity":				"Трансзакции",
+	"tabs.home.activity.empty":	"Нет активности",
+	"tabs.home.empty":					"Пришлите 20 XLM для активации аккаунта.",
```

Comment:
Please change this to a more general message so that it doesn't need to be re-done each time that the reserve_base value on the Stellar network changes.  Thanks!